.coveragerc

```diff
@@ -529,6 +529,11 @@ omit =
     homeassistant/components/metoffice/sensor.py
     homeassistant/components/metoffice/weather.py
     homeassistant/components/microsoft/tts.py
+    homeassistant/components/microsoft_graph/__init__.py
```

Comment:
In general we want all files tested, since it helps make the product more stable.

You have sone tests, so it is a surprise to see __init__.py and const.py marked as not tested.